### PR TITLE
Synergy patch #1

### PR DESCRIPTION
**Pull request by Viciousaurus**
Adresses:
#27417 
#27418
#27419
#27420
#27421
#27422
#27423
#27424 
#27425
#27426
#27427
#27428
#27430
#27431
#27432
#27433
#27434 

:cl:
 * rscadd: Synergy: Multiple fixes.
 * bugfix: Fixed Access issues in the Bar, Chapel, Escape, Execution chamber, AI core and some other places
 * bugfix: Removed blast doors intersection in the AI room. Experiment failed; If you want to meet it you'll have to walk the long way

* rscadd: The union won, xenobiology has slimes roundstart

* bugfix: Extra paramedic won't be sent to the cube

* bugfix: Bar sign has been fixed. Bar has more decoration instruments.

* rscadd: R&D has acid

* rscadd: Medbay now has Nanomed machines, cleaners, defibrilators. Also small medbay mapping changes.

* rscadd: Megaseed now has seeds

* rscadd: Tracking beacons added

* spellcheck: The Technical Storage is now called Technical Storage

* rscadd: Status displays added to the escape shuttle

* rscadd: Power monitors now track power.

* rscadd: Added some cameras, notably to virology maintenance

* tweak: Inky, Blinky and Pinky will not demolish the everloving shit out of everything.

* rscadd: Added petting.

### CODE REVIEW

**Comment by Viciousaurus** [comment]
This doesn't adress particular gripes with the map design, department structure and stuff. I need to see more of it in action to generate a solution/fix, if one is needed.
tl;dr play my shit map

**Comment by gurfan** [comment]
Very quick!  👍

**Comment by beesil** [comment]
Good fixes. My only gripe is that the main halls could use more o2 lockers.

**Comment by Viciousaurus** [comment]
Roger

**Comment by kernikov** [comment]
I like the supermatter, very funny.

**Comment by SonixApache** [comment]
improve :cl: and speedmerge
